estate/__manifest__.py

```diff
@@ -0,0 +1,17 @@
+{
+    'name': 'Real Estate',
+    'description': 'Real Estate !',
+    'version': '1.0',
+    'summary': 'Track Real Estate',
+    'website': 'https://www.odoo.com/app/realestate',
+    'author': 'Odoo S.A.',
+    'license': 'LGPL-3',
```

Comment:
```suggestion
    'application': True,
    'license': 'LGPL-3',
```
You probably want to have application=True for this module. See the [manifest reference](https://www.odoo.com/documentation/19.0/developer/reference/backend/module.html).